prose.md

```diff
@@ -51,169 +52,169 @@ grass, which leads to a crosswalk. There is a sign in front of the unpaved path
 
-> There is no right or wrong, only better or worse
+> There is no right or wrong, only better or worse.
 >
 > --- GW
 
-This quote by Greg Wilson was written after several iterations of the lesson
-infrastructure and I put it here as a reminder that the infrastructure we have
-right now was working for us at the time and what we come up with will have its
+Greg Wilson wrote this after several iterations of the lesson
+infrastructure. I am putting it here as a reminder that the infrastructure we have
+now was working for us at the time. What we come up with will have its
 own difficulties down the road, but what is important is that we build something
-that better addresses the needs of the community, which leads me to my second
+that better addresses the needs of our community, which leads me to my second
 reminder:
 
 > In The Carpentries: you belong
 
 As the community has grown, our infrastructure has been put to the test and we
-have continuously updated our workflows to lower barriers for people. The reason
-we do this is because our values are what drive us and in The Carpentries, you
+have continuously updated our workflows to make it easier for people to contribute.
+The reason we do this is because our values are what drive us. In The Carpentries, you
 belong, no matter if you have been working as a systems administrator for a
 university HPC cluster or if you have just learned to write your first R script.
-If you are unfamiliar with how we build our lessons, the next section is all
-about our infrastructure.
+
+To better understand the decisions we are making, let's start by reviewing
+our current infrastructure.
 
 ## Infrastructure
 
 > timing 2021-06-09: 04:20.38
 > timing 2021-06-09: 03:52.10
 > timing 2021-06-10: 03:08.87
 
-Our lessons are written in markdown and translated to HTML via GitHub pages and
+Our lessons are written in markdown and into a website via GitHub Pages and
 the Jekyll static site generator. The idea behind this choice was that it was
 the most straightforward way to create a static website without needing a server
-like wordpress or drupal sites. Ideally, it also provided a way for people to
+like Wordpress or Drupal sites. Ideally, it also provided a way for people to
 use this as an example of how they could build their own website.
 
-It's important to pause here and point out that the entire paradigm for this is
-for people to be able to write markdown lessons and get a functional website
-out of it. This is not a new concept, in fact, there are over 460 iterations of
+The paradigm of being able to write markdown lessons and get a functional website
+out of it is not a new concept. In fact, there are over 460 iterations of
 this concept according to https://staticsitegenerators.net. Jekyll happens to
-be the one that was used by GitHub early on and thus it stuck.
+be the one that GitHub implemented early on to provide documentation for the
+open-source project it hosted, and thus, it stuck.
```

Comment:
ah, good point about highlighting the _reason_ why they chose it. Is the project you are referring to github itself?